web/template/manage-pii-details.gotmpl

```diff
@@ -0,0 +1,172 @@
+{{ template "page" . }}
```

Comment:
This template needs redoing. there are a lot of injected tags, e.g. data-dashlane-id that shouldn't be there.